flink-kubernetes/src/test/java/org/apache/flink/kubernetes/highavailability/KubernetesHighAvailabilityRecoverFromSavepointITCase.java

```diff
@@ -63,12 +64,16 @@
 import java.util.Random;
 import java.util.concurrent.TimeUnit;
 
+import static org.apache.flink.kubernetes.KubernetesExtension.KUBE_CONF_ENV;
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * Tests for recovering from savepoint when Kubernetes HA is enabled. The savepoint will be
  * persisted as a checkpoint and stored in the ConfigMap when recovered successfully.
  */
+@EnabledIf(
+        value = "org.apache.flink.kubernetes.KubernetesExtension#checkEnv",
+        disabledReason = "Disabled as " + KUBE_CONF_ENV + " is not set.")
```

Comment:
A strange thing: If `@EnabledIf` is worked in `class-level`, all tests can be skipped, but the disabled reason will not be printed. 

It will display 'Process finished with exit code 0' when running in my IDE.

Reply:
Only If at the method level, it will be printed out. Similar problem seem to exist at `@Disabled("xxx")`.